src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Consider using a variable for `url(../images/star.svg)` since it is used more than once in this SCSS file. This will make it easier to change the path or the image if needed in the future.